packages/plugin-ext/src/main/browser/view/plugin-view-widget.ts

```diff
@@ -59,6 +60,7 @@ export class PluginViewWidget extends Panel implements StatefulWidget {
         const widget = this.widgets[0];
         if (widget) {
             widget.activate();
+            this.updateWidgetMessage();
```

Comment:
it should not be here, activate means focus something in already rendered widget

Reply:
`this.updateWidgetMessage` has nothing to do with `PluginViewWidget`'s rendfering. It just  "pushes" the 'message'  value further into the `TreeViewWidget` in case it is already created and added here.
The way `onActivateRequest` method is written:
```
        const widget = this.widgets[0];
        if (widget) {
            widget.activate();
        } else {
            this.node.focus();
        }
```
tells nothing about that the widget is already rendered.  Imho, it's not even initialized properly in case of `this.widgets[0]; === undefined`, is it?